e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/fee-record-correction/createFeeRecordCorrectionRequestPage.js

```diff
@@ -0,0 +1,10 @@
+const createFeeRecordCorrectionRequestPage = {
+  mainHeading: () => cy.get(`[data-cy="main-heading"]`),
```

Comment:
This should come from partials now to be DRY

Reply:
Thanks, removed this and now using the TFM partial